### PR TITLE
Add create and update tests for plan endpoint

### PR DESCRIPTION
**Pull request by jorbush**
I had problems with GET and DELETE endpoint. I've decided to delegate them to @AlbertMartinezM. Good luck 🍀

image

### CODE REVIEW

**Review by AlbertMartinezM — COMMENTED** [review]
seems great 👍 
Before starting with testing i will change the auth workflow, as we talked the other day.
(switching from JWT in Cookies to JWT in auth header.)

Won't approve this PR yet because i have to update some of the test to match with these new changes.

File: backend/resources/plan.py [hunk 1]
```diff
@@ -134,5 +134,5 @@ def delete(self, id):
         except (PlanCreationError, UserCreationError, PlanDeletingError) as e:
             return {'message': e.message}, 400
         except Exception as e:
+            print(e)
```
Comment (AlbertMartinezM) [inline backend/resources/plan.py:137]: this line should be deleted after testing 😸

Comment (jorbush) [reply backend/resources/plan.py]: Fixed! Thank u

**Comment by jorbush** [comment]
> seems great 👍 Before starting with testing i will change the auth workflow, as we talked the other day. (switching from JWT in Cookies to JWT in auth header.)
> 
> Won't approve this PR yet because i have to update some of the test to work with these new changes.

I think you are wrong. We have to merge this PR and your changes should pass these new tests.

**Review by AlbertMartinezM — APPROVED** [review]
Approved. Will change failing tests in another PR

**Pull request merged**